user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Automatic_incident_tracking.md

```diff
@@ -182,11 +182,10 @@ In addition, the following configuration is needed in the *configuration.xml* fi
   </item>
   ```
 
-After editing and saving the configuration file, **restart the SLAnalytics process on the leader agent** to make sure the desired changes take effect.
+After editing and saving the configuration file, **restart the SLAnalytics process on the leader Agent** to make sure the desired changes take effect.
 
 > [!TIP]
-> The *configuration.xml* files are not yet synced across a dataminer cluster. For this reason, it is important to always edit the file on the leader agent.
-> If possible, it is good practice to manually replace the file on all Agents in the cluster at the same time. This way, no unwanted behavior will occur when changing the leader agent at a later moment.
+> The *configuration.xml* files are not synced across a DataMiner cluster. For this reason, it is important to always edit the file on the leader Agent. If possible, it is good practice to manually replace the file on all Agents in the cluster at the same time. This way, no unwanted behavior will occur when the leader Agent is changed at a later moment.
```

Comment:
@finestracheride Is it possible to [force a synchronization of this file](https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Systems/DMS_maintenance_tools/Synchronizing_data_between_DataMiner_Agents.html#forcing-synchronization-of-a-file-with-the-dms) instead? If yes, this is probably more user-friendly than replacing the file manually.

Reply:
That would indeed be the best solution, but it is not straightforward. A task was made a long time ago, but it is not considered a priority at the moment because of the amount of work it will take and the limited use of it, being this advanced configuration quite hidden and only used in some very special cases for now.